cmd/tempo-serverless/Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: build-docker build-zip test
+
+# builds a docker image named "tempo-serverless" using the pack tool
+#   https://github.com/buildpacks/pack
+build-docker: 
+	go mod vendor
+
+	pack build \
+	  --builder gcr.io/buildpacks/builder:v1 \
+	  --env GOOGLE_RUNTIME=go \
+	  --env GOOGLE_FUNCTION_SIGNATURE_TYPE=http \
+	  --env GOOGLE_FUNCTION_TARGET=Handler \
+	  tempo-serverless
+
+	rm -rf vendor
```

Comment:
Is there a particular reason to not check in the vendor folder? We do it for our top-level modules.

Reply:
I'm honestly not sure the best way to handle this. The vendored folder actually includes tempo files b/c this module vendors Tempo. For now I think I'd rather leave them out while we figure out best practices with nested go modules.